docs/en/guides/dom-events.md

```diff
@@ -31,31 +32,31 @@ wrapper.trigger('click', {preventDefault: true})
 ```
 
 
-## Mouse Click Example
+## Exemple de clic de souris
 
-**Component under test**
+**Componsant à tester**
 
 ```js
 <template>
 <div>
-  <button class="yes" @click="callYes">Yes</button>
-  <button class="no" @click="callNo">No</button>
+  <button class="oui" @click="callOui">Oui</button>
```

Comment:
On a pris le parti de ne traduire que les chaines de caractère et les commentaire ce qui donne

```
<button class="yes" @click="callYes">Oui</button>
```

```

Reply:
Du coup tu traduirais le `this.callMe('oui')` ou tu le laisserais en `this.callMe('yes')`? 
Je prends le parti dans un premier temps de remettre `yes`.